src/20241106_Multi_GHOAvaxLaunch/AaveV3Arbitrum_GHOAvaxLaunch_20241106.sol

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {TransparentUpgradeableProxy} from 'solidity-utils/contracts/transparent-proxy/TransparentUpgradeableProxy.sol';
+import {UpgradeableBurnMintTokenPool} from 'ccip/pools/GHO/UpgradeableBurnMintTokenPool.sol';
+import {UpgradeableTokenPool} from 'ccip/pools/GHO/UpgradeableTokenPool.sol';
+import {RateLimiter} from 'ccip/libraries/RateLimiter.sol';
+import {TokenAdminRegistry} from 'ccip/tokenAdminRegistry/TokenAdminRegistry.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/src/interfaces/IProposalGenericExecutor.sol';
+import {AaveV3Arbitrum, AaveV3ArbitrumAssets} from 'aave-address-book/AaveV3Arbitrum.sol';
+import {GovernanceV3Arbitrum} from 'aave-address-book/GovernanceV3Arbitrum.sol';
+import {MiscArbitrum} from 'aave-address-book/MiscArbitrum.sol';
+import {IGhoToken} from 'gho-core/gho/interfaces/IGhoToken.sol';
+
+library Utils {
```

Comment:
probably unneeded. We put everything in one single contract

Reply:
Removed in [5e07b5a](https://github.com/bgd-labs/aave-proposals-v3/pull/519/commits/5e07b5a1496a8713d544819b1cdf632500e883ee)